qemu/tests/multi_vms_with_stress.py

```diff
@@ -95,6 +96,13 @@ def wait_for_shutdown_all_vms(vms, sessions):
     run_stress_background()
     is_stress_alive()
 
+    if params.get('set_maxcpus'):
```

Comment:
suggest change to params.get('set_maxcpus',"no")=="yes":

Reply:
thanks a lot, it solved.